.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+    "recommendations": [
+        "ms-dotnettools.csharp"
+    ]
+}
```

Comment:
Added a few files to make it easier to code on Visual Studio Code. Users would normally open the root folder to work on C# code, and `/java-library` folder for Java code.